{
  "modified": "2024-08-21T14:53:10Z",
  "published": "2024-08-21T14:53:10Z",
  "schema_version": "1.5.0",
  "id": "",
  "summary": "Malicious code in puffioner131 (PyPI)",
  "details": "The OpenSSF Package Analysis project identified 'puffioner131' @ 99999999 (pypi) as malicious.\n\nIt is considered malicious because:\n\n- The package communicates with a domain associated with malicious activity.\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "puffioner131"
      },
      "versions": [
        "99999999"
      ]
    }
  ],
  "credits": [
    {
      "name": "OpenSSF: Package Analysis",
      "type": "FINDER",
      "contact": [
        "https://github.com/ossf/package-analysis",
        "https://openssf.slack.com/channels/package_analysis"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "ossf-package-analysis",
        "sha256": "3f949c1ed3b242666f9fceeb6adf4ec7105c6f257aaa86080d0fd154fa56a80f",
        "import_time": "2024-09-18T02:42:11.382249132Z",
        "modified_time": "2024-08-21T14:53:10Z",
        "versions": [
          "99999999"
        ]
      }
    ]
  }
}
